packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -25,7 +25,7 @@ const cleanFilename = (name) =>
 	);
 
 async function clientConfig(env) {
-	const { isProd, source, src, refresh, cwd /*, port? */ } = env;
+	const { isProd, source, src, cwd /*, port? */ } = env;
```

Comment:
We should probably remove any mentions of the `--refresh` flag in the Readme too👍

Reply:
it still works, just removed from the `clientConfig` and now exclusively present in the `devConfig`